src/platform_impl/macos/app_delegate.rs

```diff
@@ -81,9 +86,13 @@ extern "C" fn dealloc(this: &Object, _: Sel) {
   }
 }
 
-extern "C" fn did_finish_launching(this: &Object, _: Sel, _: id) {
+extern "C" fn will_finish_launching(this: &Object, _: Sel, _: id) {
+  AppState::will_launch(this);
+}
```

Comment:
You should use `trace!` macro like [here](https://github.com/tauri-apps/tao/pull/445/files#diff-b0c452c19976b1833249452c2a69dcc0e578a207b9c214a1a97261967337c762R94).

Reply:
Added `trace!` !